core/lib/types/src/api/mod.rs

```diff
@@ -556,19 +556,19 @@ pub struct GetLogsFilter {
 
 /// Result of debugging block
 /// For some reasons geth returns result as {result: DebugCall}
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
-0 on such changes: they increase the diff while arguably the benefit is marginal/subjective.
When you have to read several 1k+ sized PRs, it adds up and makes reviewing harder than necessary.
Probably it'd be better to ship such changes as separate smallish PRs.